pages/project-details/[id]/index.js

```diff
@@ -105,6 +142,15 @@ export default function ProjectDetails({
             currentProject={currentProject}
           />
         </ViewsSection>
+        {currentProject.dimensions ? (
+          <Materiallist
+            addNewMaterial={addNewMaterialProject}
+            currentEntry={currentProject}
+            entries={projects}
+            handleMateriallistDelete={handleMateriallistDeleteProjects}
+            projectId={id}
```

Comment:
you pass the `id` down as a `projectId` and in your Materiallist you call your addNewMaterial with `projectId` and `newDimension`.

It looks to me like you don't need this prop since you already have access to it in this file.
After removing this prop you can also update your `addNewMaterial` to only work with `newDimension`.

Reply:
i fixed that 🙂